brave/brave-ios
# Fix #2584: No longer check clipboard if ref code exists on launch.

## Summary of Changes


This pull request fixes #2584

No longer allow ref codes to be overwritten from clipboard on subsequent launches when network request fails.

## Submitter Checklist:


## Test Plan:


  1. Copy a referral link (or retrieve from website) `F83AB73F-9852-4F01-ABA8-7830B8825993:ABC123`
  1. Disable internet
  1. Launch Brave (reads from clipboard)
  1. See ref code in settings
  1. Force quit Brave
  1. Copy new ref code (`F83AB73F-9852-4F01-ABA8-7830B8825993:XYZ789`)
  1. Re-open Brave (internet still off)
  1. Look at settings, should still see same / old ref-code
  1. Quit~~e~~ Brave 
  1. Enable internet
  1. Re-open Brave
  1. Check logs (should see some server response, either invalid code, or a download id)

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
